unit-tests/func/frames/test_first_frame_delay.py

```diff
@@ -0,0 +1,68 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+# test:device L500*
+# test:device D400*
+
+import pyrealsense2 as rs
+from rspy.stopwatch import Stopwatch
+from rspy import test
+import time
+import platform
+
+# Start depth + color streams and measure the time from stream opened until first frame arrived.
+# Verify that the time do not exceeds the maximum time allowed
+MAX_TIME_TO_WAIT_FOR_FIRST_FRAME = 2.5  # [sec]
+open_call_stopwatch = Stopwatch()
+first_frame_time = None
+
+
+def time_to_first_frame(sensor, profile):
```

Comment:
Add docs -- still unclear to me what it's supposed to return if we didn't get a frame before MAX_TIME.
Also, I'd pass the max time in as an argument and wait an extra second?

Reply:
Done